Makefile

```diff
@@ -5,9 +5,10 @@ CONFIG_SUB_REV = 3d5db9ebe860
 BINUTILS_VER = 2.44
 GCC_VER = 9.4.0
 MUSL_VER = 1.2.5
-GMP_VER = 6.1.2
-MPC_VER = 1.1.0
-MPFR_VER = 4.0.2
+GMP_VER = 6.3.0
+MPC_VER = 1.3.1
+MPFR_VER = 4.2.2
+ISL_VER = 0.27
```

Comment:
ISL_VER wasn't set by default so far. was that a mistake or a deliberate omission?

Reply:
It was originally deliberate since the ISL stuff was bleeding-edge/experimental. Nowadays maybe folks expect it to be there, I dunno. If it's to be enabled by default that should be its own documented change not silently slipped in here.